plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3/HiveS3Config.java

```diff
@@ -354,6 +355,19 @@ public HiveS3Config setS3ConnectTimeout(Duration s3ConnectTimeout)
         return this;
     }
 
+    public Duration getS3ConnectTtl()
```

Comment:
I would recommend changing to `Optional<Duration>` and add `@NotNull` annotation.